voidcrew/edits/subsystem/minor_mapping.dm

```diff
@@ -0,0 +1,7 @@
+
+/datum/controller/subsystem/minor_mapping/trigger_migration()
+	return
```

Comment:
``trigger_migration`` on roundstart is a config we can remove through ``mice_roundstart``, getting it through random event should also be a config